src/config.json

```diff
@@ -744,9 +744,17 @@
     "nfinity.space",
     "catctus.io",
     "caucus.so",
-    "exinity.com"
+    "exinity.com",
+    "markeeta.sk",
+    "markeeta.cz",
+    "cryptoknitties.io",
+    "factus.ru",
+    "etherscene.com",
+    "metamash.io",
+    "cactus.black"
   ],
   "blacklist": [
+    "amatus.capital",
```

Comment:
I'm guessing you meant to add this to the whitelist? It's already blocked at the moment, by the fuzzy list.